package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why this file is in the root level of the project?

Reply:
you right, I'm gonna change it